src/components/designSystem/NavigationTab.tsx

```diff
@@ -103,7 +110,7 @@ export const NavigationTab = ({
 
     // NOTE: window.location.pathname has to be watched for programatic navigation (without clicking on tabs)
```

Comment:
What about this comment then? Should it be updated and does this programatic url change still triggers the hook? I guess yes but just double checking as copy seems wrong now